packages/Symfony/src/Rector/Identical/ResponseStatusCodeRector.php

```diff
@@ -0,0 +1,132 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Identical;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\BinaryOp\Identical;
+use PhpParser\NodeDumper;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+final class ResponseStatusCodeRector extends AbstractRector
+{
+    /**
+     * @var array
+     */
+    private $codeToConst = [];
+
+    public function __construct()
+    {
+        $ref = new \ReflectionClass('Symfony\\Component\\HttpFoundation\\Response');
+
+        foreach ($ref->getConstants() as $key => $value) {
+            if (strpos($key, 'HTTP_') === 0) {
+                $this->codeToConst[$value] = $key;
+            }
+        }
+    }
+
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition('Turns status code numbers to constants', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+class SomeController
+{
+    public function index()
+    {
+        $response = new \Symfony\Component\HttpFoundation\Response();
+        $response->setStatusCode(200);
+        
+        if ($response->getStatusCode() === 200) {}
+    }
+}
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+class SomeController
+{
+    public function index()
+    {
+        $response = new \Symfony\Component\HttpFoundation\Response();
+        $response->setStatusCode(\Symfony\Component\HttpFoundation\Response::HTTP_OK);
+        
+        if ($response->getStatusCode() === \Symfony\Component\HttpFoundation\Response::HTTP_OK) {}
+    }
+}
+CODE_SAMPLE
+
+            )
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [Identical::class, Node\Expr\MethodCall::class];
```

Comment:
What about `NotIdentical`? `NotEqual`?...

Better use `BinaryOp` to cover any compare.